<?xml version="1.0" encoding="utf-8"?>
<launch>
    <!-- 加载轨迹规划主控 -->
    <node pkg="wtr_race" type="traj_plan_3D" name="traj_plan_3D" output="screen">
        <rosparam file="$(find wtr_race)/config/traj_params.yaml" command="load" />
    </node>
    <!-- 加载rviz配置 -->
    <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find wtr_race)/config/traj.rviz" />

    <!-- 加载二维slam地图 -->
    <include file="$(find wtr_race)/launch/wtr_map_server.launch"/>
    <!-- 加载手柄 -->
    <include file="$(find wtr_race)/launch/wtr_joy.launch" />

    <!--加载比赛车 -->
    <!-- <include file="$(find assembly2)/launch/display.launch" />      
    <include file="$(find assembly2)/launch/gazebo.launch" /> -->

</launch>